modules/aws_k8s_base/tf_module/opta_base.tf

```diff
@@ -10,6 +10,17 @@ resource "helm_release" "opta_base" {
     })
   ]
   depends_on = [
-    helm_release.cert_manager
+    time_sleep.wait_a_bit
   ]
+}
+
+resource "time_sleep" "wait_a_bit" {
```

Comment:
Let's just try by putting all these dependencies in the depends on above. We have too many sleeps in opta now. That will slow opta down a lot

Reply:
We use this exactly once and that's for gcp. Even if we did, the apply sometimes fails on the cert manager check because waiting for it to be deployed is not enough-- it needs a few more seconds to be operational